docs/content/configuration/index.md

```diff
@@ -257,6 +257,14 @@ This config is used to find the [Indexing Service](../design/indexing-service.ht
 |--------|-----------|-------|
 |`druid.selectors.indexing.serviceName`|The druid.service name of the indexing service Overlord node. To start the Overlord with a different name, set it with this property. |druid/overlord|
 
+### Broker Discovery
+
+This config is used to find the [Brokers](../design/broker.html) using Curator service discovery.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.selectors.broker.serviceName`|The druid.service name of the broker node. |druid/broker|
```

Comment:
There is also not necessarily going to be just one kind of broker per cluster. Some people have different brokers they use for different use cases.